tests/README.rst

```diff
@@ -80,40 +80,42 @@ To do that, run `podman pull quay.io/rhinstaller/anaconda-ci:master` or build
 it locally again.
 
 
-Explanation of the GitHub configuration
----------------------------------------
+GitHub workflows
+----------------
+
+All test and maintenance actions are run by `GitHub workflows`_.  These YAML
+files completely describe what steps are required to run some action, what are
+its triggers and so on.
 
 Pull request for master:
 ________________________
 
-Everything is stored in the `.github/workflows/validate.yml`_ workflow.
-These files completely describe what steps are required to run the tests, what
-are the triggers and so on. We use GitHub runners here so we don't have to care
-too much about what is executed there. GitHub runners are designed to be isolated
-and secure. On master we are building the container only if the container files
-have changed, otherwise we are pulling the container from our repository. For more
+Unit and rpm tests are run by the `validate.yml workflow`_.  We use GitHub's
+runners for this so we don't have to care about what is executed there.
+
+The workflow rebuilds the ``anaconda-ci`` container if the container files
+have changed, otherwise it is pulling the container from quay.io. For more
```

Comment:
I would make a link to our repository from the ``quay.io``.

Reply:
Updated.